Add severity level to Deployment health incidents (#3345)
astronomer/docs: astro/deployment-health-incidents.md

![A Deployment Health incident message appearing after a user hovers over the Deployment health status](/img/docs/incident-unfurl.png)

> The following table contains all types of Deployment incidents.

Author: ```suggestion
The following table contains all types of Deployment incidents. A **warning** incident indicates that specific tasks or DAGs might fail, while a **critical** incident indicates that your entire Deployment might not work as expected.
```

| Incident name                           | Severity | Description                                                                                        |
| --------------------------------------- | -------- | -------------------------------------------------------------------------------------------------- |
| Scheduler Heartbeat Not Found           | Critical | The Airflow scheduler has not sent a heartbeat for longer than expected.                           |
| Airflow Database Storage Unusually High | Warning  | The metadata database has tables that are larger than expected.                                    |
| Worker Queue at Capacity                | Warning  | At least one worker queue in this Deployment is running the maximum number of tasks and workers.   |
| Worker Queue Does Not Exist             | Warning  | There is at least 1 task instance that is configured to run on a worker queue that does not exist. |

> A **warning** incident indicates that specific tasks or DAGs might fail, while a **critical** incident indicates that your entire Deployment might not work as expected. Use the following topics to address each of these incidents. 

Author: ```suggestion
Use the following topics to address each of these incidents. 
```

### Scheduler Heartbeat Not Found 

> The scheduler has not sent a heartbeat for longer than 30 seconds. This could be a sign that the scheduler is down. Tasks will keep running, but new tasks will not be scheduled. 

Author: ```suggestion
The scheduler has not sent a heartbeat for longer than 10 minutes. This could be a sign that the scheduler is down. Tasks will keep running, but new tasks will not be scheduled. 
```
